pkg/crc/errors/multierror.go

```diff
@@ -22,12 +22,30 @@ func (m MultiError) ToError() error {
 	if len(m.Errors) == 0 {
 		return nil
 	}
+	if len(m.Errors) == 1 {
+		return m.Errors[0]
+	}
+
+	var aggregatedErrors []string
 
-	errStrings := []string{}
-	for _, err := range m.Errors {
-		errStrings = append(errStrings, err.Error())
+	count := 1
+	current := m.Errors[0].Error()
+	for i := 1; i < len(m.Errors); i++ {
```

Comment:
Sorry, not going to be a real review, but more a list of design questions
Wondering if  `for i, err := range(m.Errors) {` could be used somehow? (I know you to skip the first item).
Would an `AggregatedError` type be better than `errorWithCount` returning just a string? This could  even implement `Unwrap()` ( https://blog.golang.org/go1.13-errors )

Why isn't `Multierror` implementing `error` directly rather than creating a fake error with a flattened string?

Reply:
I removed the ToError() function and now MultiError is a proper error. For the Unwrap() function, I did nothing. Maybe in follow up?

Reply:
Yeah, this really was more random ramblings where you can pick and choose what makes sense than a list of things which must be done right now :)